catalog/MDCCatalog/MDCCatalogTiles.h

```diff
@@ -50,3 +50,4 @@ void MDCCatalogDrawSwitchTile(CGRect frame, NSObject<MDCColorScheme> *colorSchem
 void MDCCatalogDrawTabsTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme);
 void MDCCatalogDrawTextFieldTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme);
 void MDCCatalogDrawTypographyTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme);
+
```

Comment:
Please revert the formatting change to this file.
(Cleanup is welcome, but we like to keep them in their own PRs)